extensions/observability-devservices/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Observability"
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+metadata:
+  keywords:
+    - "lgtm"
+    - "grafana"
+    - "prometheus"
+    - "tempo"
+    - "opentelemetry"
+    - "tracing"
+    - "distributed-tracing"
+    - "monitoring"
+  categories:
+    - "observability"
+  config:
+    - "quarkus.observability."
+  status: "experimental"
+  guide: "https://quarkus.io/guides/observability-devservices"
```

Comment:
The guide should point directly to: https://quarkus.io/guides/observability-devservices-lgtm